core/spec.md

```diff
@@ -5071,7 +5090,7 @@ When converting a cross-reference Resource back into a "normal" Resource, the
 following MUST be adhered to:
 - The request MUST delete the `xref` attribute or set it to `null`.
 - A default Version of the Resource MUST be created.
-- Any attributes specified at the Resource level MUST be interpreted as a
+- Any attributes specified at the Resource-level MUST be interpreted as a
```

Comment:
I think this hyphen is incorrect, personally. If it were "Any Resource-level attributes" that would be okay, but with the current structure I think the space was better.